buildscripts/builddrivers.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
```

Comment:
the point might be moot in windows as I think this is ignored. But you said this requires python 3.4 so it should be /usr/bin/python3